models/demos/ttnn_resnet/tt/ttnn_functional_resnet50_new_conv_api.py

```diff
@@ -632,15 +657,38 @@ def __init__(
 
         conv_dummy_tensor = torch.rand((self.fold_output_shape), dtype=torch.bfloat16)
         conv_dummy_tensor = ttnn.from_torch(conv_dummy_tensor, layout=ttnn.ROW_MAJOR_LAYOUT)
-        _, self.override_fold_mem_config, _, _ = ttnn.get_conv_padded_input_shape_and_mem_config(
-            device=device,
-            input_tensor=conv_dummy_tensor,
-            conv_config=self.conv1_config,
+
+        parallel_config = ttnn._ttnn.operations.conv.determine_parallel_config(
+            shard_layout=ttnn.TensorMemoryLayout.HEIGHT_SHARDED,
             batch_size=self.batch_size,
-            height=self.conv1_output_height,
-            width=self.conv1_output_width,
-            in_channels=self.conv1_input_channels,
-            out_channels=self.conv1_output_channels,
+            input_channels=self.conv1_input_channels,
+            output_height=self.conv1_output_height,
+            output_width=self.conv1_output_width,
+            output_channels=self.conv1_output_channels,
+            compute_grid_size=device.compute_with_storage_grid_size(),
+            block_shard_orientation=ttnn.ShardOrientation.ROW_MAJOR,
+            enable_channels_padding=True,
+            is_out_tiled=True,
+        )
+        # Override compute grid size for Grayskull
+        # First convs would got to 108 cores by default
+        # but this would add padding into output tensor
+        # and reshard that follows first conv fails with padding ATM.
+        if is_grayskull():
+            compute_grid = device.compute_with_storage_grid_size()
+            parallel_config.grid = get_core_grid_from_num_cores(98, compute_grid.x, compute_grid.y)
```

Comment:
Maybe mention 98 or a reduction of 10 in the comment.

Also, please I don't know what you mean by first convs and the grammar for "would got" is not correct.